Samples/UniversalDecoder/tools/copy-codecs.js

```diff
@@ -25,9 +28,18 @@ glob.sync(`**/*`,
     fse.copySync(srcPath, dstPath);
 
     if (f.endsWith(".js")) {
-      fs.appendFile(dstPath, '\nmodule.exports={decodeUplink};', function (err) {
-        if (err) throw err;
-        console.log(`Patching ${dstPath}`);
-      });
+      const tree = esprima.parseScript(fs.readFileSync(srcPath).toString());
+      if (tree.body.find(n => n.type === 'FunctionDeclaration' && n.id.name === 'decodeUplink')) {
```

Comment:
nit: a comment explaining the logic here would help

Reply:
Added with 2515958db46883d17e2b936cda4ca3b7a5792b0f.